### PR TITLE
Throw NotFoundHttpException when debug is disabled

### PR DESCRIPTION
**Pull request by salkhwlani**
Throw NotFoundHttpException when debug is disabled and pass string in
URL

for example `http://site.com/user/1844379023` work fine but
`http://site.com/user/test` throw InvalidArgumentException

### CODE REVIEW

File: src/FakeIdServiceProvider.php [hunk 1]
```diff
@@ -69,7 +69,11 @@ protected function registerRouterMacro()
 
                 // Decode FakeId first if applicable.
                 if (in_array('Propaganistas\LaravelFakeId\FakeIdTrait', class_uses_recursive($class))) {
-                    $value = $this->container->make('fakeid')->decode($value);
+                    try {
+                        $value = $this->container->make('fakeid')->decode($value);
+                    } catch (\InvalidArgumentException $e) {
+                        throw config('app.debug') ? new  \InvalidArgumentException($e->getMessage()) : new NotFoundHttpException;
```
Comment (Propaganistas) [inline src/FakeIdServiceProvider.php]: No need to reconstruct an InvalidArgumentException. Just rethrow `$e`:

```php
throw config('app.debug') ? $e : new NotFoundHttpException;
```

**Review by Propaganistas — CHANGES_REQUESTED** [review]
Thanks. Good addition. Please process my feedback. Could you also add tests?

**Review by salkhwlani — COMMENTED** [review]
you right, check now

**Review by Propaganistas — APPROVED** [review]

**Comment by Propaganistas** [comment]
The tests are failing. Please fix and also revert the tests file structure & Route facade use.

**Comment by Propaganistas** [comment]
Thanks. A new release has been tagged: `2.0.6`

**Pull request merged**